arches/app/models/graph.py

```diff
@@ -563,22 +563,18 @@ def delete(self):
                 )
             )
 
-    def delete_instances(self, verbose=False):
+    def delete_instances(self, verbose=False, request=None):
```

Comment:
Why is this an optional argument if we use it unconditionally? Can we just make the user id the argument?